prime-dotnet-webapi/Models/BusinessEventType.cs

```diff
@@ -18,6 +18,7 @@ public sealed class BusinessEventType : ILookup<int>
         public const int ADMIN_VIEW_CODE = 7;
         public const int ORGANIZATION_CODE = 8;
         public const int PHARMANET_API_CALL_CODE = 9;
+        public const int PAPER_ENROLMENT_LINK_CODE = 10;
```

Comment:
These have been updated on Dev, they are now `CamelCase` rather than `SCREAMING_SNAKE_CASE`

Reply:
merged code from Dev and fixed